.env.example

```
@@ -1,12 +1,11 @@
 # Database
-DATABASE_URL="postgresql://postgres.cmvqbjbohfloobgrcdoo:your-password@aws-1-ap-northeast-1.pooler.supabase.com:6543/postgres?pgbouncer=true"
-DIRECT_URL="postgresql://postgres.cmvqbjbohfloobgrcdoo:your-password@aws-1-ap-northeast-1.pooler.supabase.com:5432/postgres"
+DATABASE_URL='postgresql://username:password@ep-xxx.region.aws.neon.tech/neondb?sslmode=require'
+DIRECT_URL='postgresql://username:password@ep-xxx.region.aws.neon.tech/neondb?sslmode=require'
+
+# NextAuth.js
+NEXTAUTH_URL=http://localhost:3000
+NEXTAUTH_SECRET='+To2Ilptzd8rpk/sgu9azKpOBGhzlwQs/KVpBORL4co='
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXTAUTH_SECRET key should go before the NEXTAUTH_URL key

(UnorderedKey)